src/stellar/stellarPlugin.js

```diff
@@ -61,6 +61,12 @@ export class StellarPlugin extends CurrencyPlugin {
     }
   }
 
+  importPrivateKey (privateKey: string): Promise<{ stellarKey: string }> {
+    privateKey.replace(/ /g, '')
+    if (privateKey.length !== 56) throw new Error('Private key wrong length')
+    return Promise.resolve({ stellarKey: privateKey })
```

Comment:
Please validate the key using `stellarApi.Keypair.fromSecret(stellarKey)`. This will detect all kinds of mathematical problems, and probably string-formatting problems too.